### PR TITLE
Bump to v0.8.5

### PR DESCRIPTION
**Pull request by quintusdias**
```
Relax dependency on lxml; use stdlib ElementTree if necessary.
Fix XML box processing bug on certain XML declarations.
Qualified on OPENJPEG v2.1.1.
```

### CODE REVIEW

**Comment by Cadair** [comment]
@jakirkham any idea why this is failing? The build seems to be ok but the upload looks like it is erroring?

@quintusdias Thanks for doing this :)

**Comment by jakirkham** [comment]
Probably need to re-render the feedstock. Please follow [these directions](https://github.com/conda-forge/conda-smithy/blob/6a55ee6b813a0516a13d341b4af3ea3d89754ea6/README.md#re-rendering-an-existing-feedstock).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+{% set version = "0.8.6rc1" %}
 package:
   name: glymur
   version: {{ version }}
 
 source:
   fn: Glymur-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/5c/8f/5633be85458c3b519262a36b10b3b9ea274acfc6d21945d005bef455093e/Glymur-0.8.4.tar.gz
-  md5: 8185ffeb8d11d1f9c14f179b581e7d96
+  url: https://pypi.python.org/packages/dc/d6/dfb8321c2bbb6a70b6e283410a61fd65b9df21245aace9fc1c6449748eaf/Glymur-0.8.6rc1.tar.gz#md5=2662194021fb219c3edb37850be81285
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please fix this URL based on the [example](https://github.com/conda-forge/staged-recipes/blob/4802ec3ffcbd04657db6bae6955d98b1892a213e/recipes/example/meta.yaml#L18).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  md5: 2662194021fb219c3edb37850be81285
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please use the sha256 checksum provided by PyPI.

**Comment by jakirkham** [comment]
Toggling to re-run CIs.

Note to self: The linter correctly handled this case and did not raise a merge conflict thanks to PR ( https://github.com/conda-forge/conda-forge-webservices/pull/68 ).

**Pull request merged**